sdk/go/command/local/command.go

```diff
@@ -99,10 +100,7 @@ func NewCommand(ctx *pulumi.Context,
 		args = &CommandArgs{}
 	}
 
-	replaceOnChanges := pulumi.ReplaceOnChanges([]string{
-		"triggers[*]",
-	})
```

Comment:
This looks like a bug as well. The schema hasn't changed, so we should see the same `replaceOnChanges` behavior here.